sesame/src/main/scala/org/w3/banana/sesame/SesameModule.scala

```diff
@@ -2,7 +2,7 @@ package org.w3.banana.sesame
 
 import org.openrdf.repository.RepositoryConnection
 import org.w3.banana._
-import org.w3.banana.io.{SparqlQueryResultsReader, _}
+import org.w3.banana.io.{ SparqlQueryResultsReader, _ }
```

Comment:
b-rdf uses both styles but I believe the original is the generally preferred style (note: not __my__ preference)